src/translator/translation_model.cpp

```diff
@@ -81,6 +85,14 @@ void TranslationModel::loadBackend(size_t idx) {
       scorer->setShortlistGenerator(shortlistGenerator_);
     }
   }
+
```

Comment:
> 4. What is the model type?

(See block above.)

Model type is whatever loaded by options. This means I can have multiple types of models at the same time (considering transformer vs transformer_s2s vs s2s sort) I'm not sure if two places (worker, model) configuring device might cause conflicts due to how state being trace-able only with significant effort.